src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Square extends Default {
+    private double size;
+
+    public Square(String color,double size) {
+        super(color);
+        this.size = size;
+    }
+
+    @Override
+    public double getProperty() {
```

Comment:
The method name 'getProperty' is not very descriptive. Consider renaming it to something more specific like 'getPerimeter' to better reflect its purpose.